### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jayunit100**
Make it clear that for none, you must be running standard docker.

### CODE REVIEW

**Comment by afbjorklund** [comment]
It would be better to fix this bug, than to document it...

minikube should respect the -`-container-runtime`

File: README.md [hunk 1]
```diff
@@ -124,7 +124,8 @@ We also released a Debian package and Windows installer on our [releases page](h
 ## Quickstart
 
 Here's a brief demo of Minikube usage.
-If you want to change the VM driver add the appropriate `--vm-driver=xxx` flag to `minikube start`. Minikube supports
+- If you want to change the container runtime, network details, consult notes from your container runtime provider.
```
Comment (RA489) [inline README.md]: AFAIK whatever may be the container runtime minikube should use it.

**Comment by afbjorklund** [comment]
Added #3295

**Comment by afbjorklund** [comment]
Added #3303 to do it properly instead

**Pull request merged**